lookout/style/format/langs/__main__.py

```diff
@@ -220,11 +220,20 @@ def analyze_parquet_row(row: pandas.Series, filepath):
     if errors:
         return 1
     reserved.discard("")
-    log.info("Internal roles: %d", len(roles))
-    log.info("UAST roles: %d", len(uast_roles))
+    log.info("Internal roles: %d", len(internal_types))
+    log.info("UAST roles: %d", len(roles))
     log.info("Reserved: %d", len(reserved))
-    uast_roles = {bblfsh.role_name(role_id): n for role_id, n in uast_roles.items()}
-    generate_files(args.output, roles, uast_roles, reserved)
+
+    sum_roles = sum(roles.values())
+    sum_internal_types = sum(internal_types.values())
+    roles = {bblfsh.role_name(role_id): n for role_id, n in roles.items()}
+    spaces = max(max(len(itype) for itype in internal_types),
+                 max(len(role) for role in roles))
+    roles = [(role, "%s%.5E" % (" " * (spaces - len(role)), n / sum_roles))
```

Comment:
You should calculate it inside Jinja2. Very easy:

```
{% set max_role_len = roles | length | max %}
{{ "%-*s" | format(max_role_len, role) }}
```

Reply:
Same for the floats, sums, etc.
```
roles.values() | sum
```
or https://github.com/pallets/jinja/issues/192